src/app/shared/components/draggable-shelf/draggable-shelf.component.html

```diff
@@ -0,0 +1,6 @@
+<div [style.flex-basis.px]="dragOffsetInPixels" [style.height.px]="dragOffsetInPixels">
```

Comment:
You can use host bindings to remove this div.

Reply:
I have done so now! :-)